podcasts/SharingHelper+swipeButton.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+import PocketCastsDataModel
+
+extension SharingHelper {
+    func shareLinkTo(episode: Episode, fromController: UIViewController, fromTableView tableView: UITableView, at indexPath: IndexPath) {
+        let source = tableView.swipeButton(forLabel: L10n.share, at: indexPath) ?? tableView
+        Analytics.track(.podcastShared, properties: ["type": "episode", "source": "episode_swipe_action"])
+        shareLinkTo(episode: episode, shareTime: 0, fromController: fromController, sourceRect: source.bounds, sourceView: source)
+    }
+}
+
+private extension UITableView {
+    func swipeButton(forLabel label: String, at indexPath: IndexPath) -> UIView? {
```

Comment:
This is terrible, I know.

However, SwipeCellKit doesn't offer any way to retrieve the view to be provided as a source.